tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,404 @@
+
+```sh
+$ simd query group group-members $GROUP_ID
+```
+
+Nice! Your group has `best football association` as metadata (which you can recall with the `group-info` command), Alice as group admin, and Alice and Bob as group members.
+
+## Manage group members
+
+To update the group's members, you send a transaction using the `update-group-members` command and a JSON file modeled on the previous `members.json`. The file only needs to contain the changes to the membership. Unchanged members do not need to be included in the `members_updates.json`.
+
+To add a member to the group, you mention it in the JSON, and to remove a member from the group, you mention it and set this member's voting weight to `0`.
+
+Let's add Carol, Dave and Emma as group members and remove Bob. Create `members_update.json` with:
+
+```json
+{
+    "members": [
+        {
+            "address": "bobaddr", // $BOB
+            "weight": "0" // this deletes bob
+            // The metadata does not need to be mentioned
+        },
+        {
+            "address": "caroladdr",
+            "weight": "1",
+            "metadata": "treasurer"
+        },
+        {
+            "address": "daveaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+                {
+            "address": "emmaaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+    ]
+}
+```
+
+```sh
+$ simd tx group update-group-members $ALICE $GROUP_ID members_updates.json
+```
+
+You can verify that the group members are updated:
+
+```sh
+$ simd query group group-members $GROUP_ID
+```
+
+As an exercise, please add Bob back in the group and go to the next section.
+
+## Create a group policy
+
+Next you need to decide on a group policy. This defines how long a proposal can be voted on and how its outcome is calculated.
+Here you use the [`ThresholdDecisionPolicy`](https://github.com/cosmos/cosmos-sdk/blob/release/v0.46.x/proto/cosmos/group/v1/types.proto#L53-L62). It defines the threshold that the tally of weighted _yes_ votes must reach in order for a proposal to pass. Each member's vote is weighted by its weight as defined in the group.
+
+Following is the content of the `policy.json`. It states that:
+
+* A proposal can be voted on for a maximum of 10 minutes.
+* A proposal requires only one vote to pass.
+
+```json
+{
+    "@type": "/cosmos.group.v1.ThresholdDecisionPolicy",
+    "threshold": "1",
+    "windows": {
+        "voting_period": "10m",
+        "min_execution_period": "0s"
+    }
+}
+```
+
+Have the group administrator create the group policy with metadata that identifies it as one with a quick turnaround:
+
+```sh
+$ simd tx group create-group-policy $ALICE $GROUP_ID "quick turnaround" policy.json
+```
+
+Check and verify your newly created group policy and in particular the address you just extracted:
```

Comment:
You did not want to also include the address extraction from the indexed transaction?

Reply:
Added